### PR TITLE
generate collection: document, taskExecution, call server compile and…

### PR DESCRIPTION
**Pull request by oma378501**
… show output.

### CODE REVIEW

File: server/methods/utility/server.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+                    console.error('error:', err || httpResponse.body);
+
+                } else {
+
+                    taskExecution.status = 0; //success
+
+                    TaskExecutions.insert(taskExecution, function(error, taskExecutionId){
+                        if (error) {
+                            $("#error").val('');
```
Comment (dmeneses) [inline server/methods/utility/server.js]: I think this not going to work on the server side, please drop.

File: server/methods/utility/server.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+                    sourceCode: documentId
+                };
+
+                if (err || httpResponse.statusCode !== 200) {
+
+                    taskExecution.status = 1; // fail
+
+                    TaskExecutions.insert(taskExecution, function (error, taskExecutionId) {
+                        if (error) {
+                            $("#error").val('');
```
Comment (dmeneses) [inline server/methods/utility/server.js]: I think this not going to work on the server side, please drop.

Comment (dmeneses) [inline server/methods/utility/server.js]: Maybe you can set in your task execution output the error received.

File: server/methods/utility/server.js (hunk 1 above)
Comment (oma378501) [inline server/methods/utility/server.js]: ok, this comment was delete.

**Pull request merged**